Functions/InputOutput/xASL_io_ReadJson.m

```diff
@@ -55,5 +55,15 @@
 	error('%s',ME.getReport()); % Display the error message
 end
 
+% In case we have a multi-context import and there is equivalent information in all contexts, it will decode each context as a struct instead of a cell
```

Comment:
Please improve comment to

In case we have a multi-context studyPar.json which ExploreASL uses for import, and there is equivalent information in all studyPar.json studies (aka "contexts"), Matlab's jsonread function will decode each context as a struct instead of a cell. Here, we bugfix this to create a cell instead.